plugins/wasi_nn/wasinnfunc.cpp

```diff
@@ -107,7 +107,6 @@ WasiNNLoadByName::bodyImpl(const Runtime::CallingFrame &Frame, uint32_t NamePtr,
   }
 
   // Get the name of model
-  std::vector<Span<uint8_t>> Builders;
```

Comment:
Why do we remove this? Is it not used anymore?

Reply:
Yes, I just help @CaptainVincent to remove this unused Builders.